examples/web_worker_fib/src/app.rs

```diff
@@ -0,0 +1,71 @@
+use crate::agent::{Worker, WorkerInput, WorkerOutput};
+
+use web_sys::HtmlInputElement;
+use yew::prelude::*;
+use yew_agent::{Bridge, Bridged};
+
+pub struct Model {
+    clicker_value: u32,
+    input_ref: NodeRef,
+    worker: Box<dyn Bridge<Worker>>,
+    fibonacci_output: String,
+}
+
+pub enum Message {
+    Click,
+    RunWorker,
+    WorkerMsg(WorkerOutput),
+}
+
+impl Component for Model {
+    type Message = Message;
+    type Properties = ();
+
+    fn create(ctx: &Context<Self>) -> Self {
+        let worker = Worker::bridge(ctx.link().callback(Self::Message::WorkerMsg));
+
+        Self {
+            clicker_value: 0,
+            input_ref: NodeRef::default(),
+            worker,
+            fibonacci_output: String::from("Try out some fibonacci calculations!"),
+        }
+    }
+
+    fn update(&mut self, _ctx: &Context<Self>, msg: Self::Message) -> bool {
+        match msg {
+            Self::Message::Click => {
+                self.clicker_value += 1;
+            }
+            Self::Message::RunWorker => {
+                if let Some(input) = self.input_ref.cast::<HtmlInputElement>() {
+                    // start the worker off!
+                    self.worker.send(WorkerInput { n: input.value_as_number() as u32 });
```

Comment:
```suggestion
                    self.worker.send(WorkerInput { 
                        n: input.value_as_number() as u32,
                    });
```
oops let me try that again 🙃